File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice7.py
T=[None for i in range(1000)]
i=0
while True:
    T[i]=float(input(f"saisir le nombre numero {i+1}: "))
    i+=1
    if(i==1000 or T[i-1]==0):
        break 
max=T[0]
max_pos=0
for j in range(0,i):
    if T[j]>=max :
        max=T[j]
        max_pos=j+1
print("le plus grand parmi ces nombres est: ",max)
print("et sa position est: ",max_pos)

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_3/exercice7.c
#include <stdio.h>
int max_pos,i;
float max,T[1000];
int main(){
    i=0;
    do{
        printf("saisir le nombre numero %d : ",i+1);
        scanf("%f",&T[i]);
        i++;
    }while(i<1000 && T[i-1]!=0);
    max=T[0];
    max_pos=0;
    for(int j=0;j<i;j++){
        if(T[j]>=max){
            max=T[j];
            max_pos=j+1;
        }
    }
    printf("le plus grand parmi ces nombres est: %f\n",max);
    printf("et sa position est: %d\n",max_pos);
    return 0;
}